app/assets/javascripts/components/story/StoryDescriptionIcon.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StoryDescriptionIcon = ({ description }) => {
+  if (description) {
+    return (
+      <span className='Story__description-icon' >
+        <i className={`mi md-dark md-16 question_answer`}>question_answer</i>
```

Comment:
What about making the icon name dynamic for this component i.e. receiving it through props? :)

Reply:
That would be easier if the class name will always be the same as the icon name, otherwise you would have to receive two more parameters and in that case it would probably be better to just keep this component as it is haha

Reply:
I think this is unnecessary since this icon is just for this specific case of story description :).